### PR TITLE
Update chicago-fullnote-bibliography-fr.csl

### PR DESCRIPTION
**Pull request by apinea**
Style was edited to reflect changes made in most recent edition of Chicago (17th).
Changes recommended and verified by Pascal Martinolli, University of Montreal librarian.

[Feuille.synthese.des.changements.a.faire.dans.le.style.CSL.pour.Zotero.docx](https://github.com/citation-style-language/styles/files/3588346/Feuille.synthese.des.changements.a.faire.dans.le.style.CSL.pour.Zotero.docx)

### CODE REVIEW

**Comment by adam3smith** [comment]
My apologies -- this and your other style are at the top of my ToDo list and I'll get to a full review and hopefully merge this week; hoping for tomorrow (Tuesday).

File: chicago-fullnote-bibliography-fr.csl [hunk 1]
```diff
@@ -594,34 +678,62 @@
...
+        <group delimiter=" ">
+          <text term="presented at" text-case="capitalize-first"/>
+          <text variable="event"/>
+        </group>
+      </else>
+    </choose>
+  </macro>
+  <macro name="originally-published">
+    <group delimiter=", ">
+      <group delimiter="&amp;#160;: ">
```
Comment (adam3smith) [inline chicago-fullnote-bibliography-fr.csl]: I'm pretty sure you want `&#160;` here and in all other instances where you use it. This is true both for this and the author-date style

**Comment by adam3smith** [comment]
I have the same comment about et-al here as with the other style -- I'm pretty sure it shouldn't be in italics. Also, please fix the non-breaking spaces to `&#160;` throughout

**Comment by apinea** [comment]
@adam3smith Thanks for your help. The recommended changes have been made to this newly updated version.

**Comment by adam3smith** [comment]
Terrific, thanks, and sorry for the initial delay

**Pull request merged**